### ISSUE TITLE
 The default displayed progress bar and the position after starting playback are inconsistent! 

### ISSUE THREAD

**Issue by AnonymHK** [zezo357/flutter_meedu_videoplayer#165]
The playback progress bar will move its position after starting playback! I tried to find the problem, but failed! The issue with the playback progress bar at the bottom of the M3u8 with qualities example is very obvious! When the video content is within 15 seconds, the progress bar becomes a dot by dot interval.... Please help correct it!

**Comment by zezo357** [comment]
Are you using latest version? The dot by dot should be fixed

**Comment by AnonymHK** [comment]
Yes, I am using the latest version. I discovered this issue a long time ago, but I overlooked it. Now, some users have reported this issue, so I am trying to fix it! On the Android platform, you can see an example of running m3u8 directly!

**Comment by zezo357** [comment]
>  Yes, I am using the latest version. I discovered this issue a long time ago, but I overlooked it. Now, some users have reported this issue, so I am trying to fix it! On the Android platform, you can see an example of running m3u8 directly! 
> 
> ![GIF 2023-8-21 星期一 4-01-36](https://github.com/zezo357/flutter_meedu_videoplayer/assets/110597435/e64319bb-7c93-4403-9b91-78465f7a0912)
> 

Ok I get it now, I will see what causes the shifting

**Comment by AnonymHK** [comment]
I suggest that the progress bar in the middle be displayed by default! Otherwise it would be a bit abrupt! After obtaining the video duration information, update the left and right duration information!

**Comment by AnonymHK** [comment]
Can you solve the problem of abnormal progress bars caused by videos within 15 seconds in length?

**Comment by zezo357** [comment]
Yes I think that's the cause of the shifting it self since that is the only thing not rendered in the above video, you can make a pr if you would like, and if not I will try to do it next weekend

**Comment by AnonymHK** [comment]
I'm trying to fix it!

**Comment by zezo357** [comment]
new release is fixed in it
